client/components/jetpack/scan-threats/index.tsx

```diff
@@ -125,87 +181,124 @@ const ScanThreats = ( { error, site, threats }: Props ) => {
 		[ updatingThreats ]
 	);
 
+	const maxSeverity = threats.reduce( ( max, threat ) => Math.max( max, threat.severity ), 0 );
+	const countMap = {
+		low: threats.filter( ( threat ) => threat.severity < 3 ).length,
+		high: threats.filter( ( threat ) => threat.severity >= 3 ).length,
+	};
+	const countMapFixable = {
+		low: allFixableThreats.filter( ( threat ) => threat.severity < 3 ).length,
+		high: allFixableThreats.filter( ( threat ) => threat.severity >= 3 ).length,
+	};
+
+	const headerSummary = getThreatCountMessage(
+		countMap.high,
+		countMap.low,
+		null,
+		translate( 'found' )
+	);
+	const fixSummary = getThreatCountMessage(
+		countMapFixable.high,
+		countMapFixable.low,
+		translate( 'Jetpack can auto fix' ),
+		null
+	);
+
+	let headerMessage = translate( 'Your site is at risk' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Your site is at risk.' )` *ES Score: 10*